src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1644,19 +1644,22 @@ public void load() {
                 return;
             }
             boolean allOk = true;
-            for (AbstractProject<?,?> job: Jenkins.getInstance().getAllItems(AbstractProject.class)) {
-                File jobCredentials = new File(job.getRootDir(), "subversion.credentials");
-                if (jobCredentials.isFile()) {
-                    try {
-                        new PerJobCredentialStore(job).migrateCredentials(this);
-                        job.save();
-                        if (!jobCredentials.delete()) {
-                            LOGGER.log(Level.WARNING, "Could not remove legacy per-job credentials store file: {0}", jobCredentials);
+            Jenkins instance = Jenkins.getInstance();
+            if (instance != null) {
```

Comment:
This approach causes a huge diff.
I would check Jenkins instance and pass empty array to the cycle if it is null